woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-token.php

```diff
@@ -721,8 +734,6 @@ public function save() {
 			}
 		}
 
-		$token->apply_changes();
```

Comment:
@unfulvio do you remember why we added `apply_changes()` in https://github.com/skyverge/wc-plugin-framework/commit/9848b634ea78917a7f73394080969827e3210ecf?

I found out that the token data store relies on pending changes to decide what information to update on the database. If we apply changes before calling save(), changed props are not persisted on update.

See [here](https://github.com/woocommerce/woocommerce/blob/e39db20eca4bbcf1b8b12c456393448f76ec8aa8/includes/data-stores/class-wc-payment-token-data-store.php#L95-L103).

Reply:
No worries. When I reviewed that change it made sense to me as well. I wasn't aware of the way the data store uses those changes.